### PR TITLE
Feature: Social login

### PR DESCRIPTION
**Pull request by hofong428**
Feature: Social login

### CODE REVIEW

**Comment by hofong428** [comment]
Here  can write some details.....

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello
```
Comment (codebat-Raymond) [inline file3.txt:1]: This line is good but it's better if you capitalize H.

**Review by codebat-Raymond — CHANGES_REQUESTED** [review]
Please make these changes and do another push.

File: file3.txt (hunk 1 above)
Comment (codebat-Raymond) [inline file3.txt:1]: That's good!

**Review by codebat-Raymond — APPROVED** [review]
Everything looks perfect. Let's ggo ahead with merging.

**Pull request merged**